src/sentry/static/sentry/app/components/errors/detailedError.jsx

```diff
@@ -50,6 +64,12 @@ class DetailedError extends React.Component {
 
               {!hideSupportLinks && (
                 <div className="detailed-error-support-links">
+                  {window.Raven &&
+                    window.Raven.lastEventId() && (
+                      <a onClick={this.openFeedback} href="">
```

Comment:
Can you use `<Button priority="link">` here instead of having an anchor w/ empty `href`